tests/E2E/__snapshots__/stylesheets.test.ts.snap

```diff
@@ -1,143 +1,94 @@
+".styles_display_flex_mobile__3nw5tz2 {
   display: flex;
 }
-.styles_display_none_mobile__3nw5tz5 {
+.styles_display_none_mobile__3nw5tz6 {
   display: none;
 }
-.styles_display_block_mobile__3nw5tz9 {
+.styles_display_block_mobile__3nw5tza {
   display: block;
 }
-.styles_paddingTop_small_mobile__3nw5tzd {
+.styles_paddingTop_small_mobile__3nw5tze {
   padding-top: 10px;
 }
-.styles_paddingTop_medium_mobile__3nw5tzh {
+.styles_paddingTop_medium_mobile__3nw5tzi {
   padding-top: 20px;
 }
-.styles_background_red_mobile__3nw5tzl {
+.styles_background_red_mobile__3nw5tzm {
   --alpha__3nw5tz0: 1;
   background: rgba(255, 0, 0, var(--alpha__3nw5tz0));
 }
-.styles_backgroundOpacity_1_mobile__3nw5tzp {
+.styles_backgroundOpacity_1_mobile__3nw5tzq {
   --alpha__3nw5tz0: 1;
 }
-.styles_backgroundOpacity_0\\\\.1_mobile__3nw5tzt {
+.styles_backgroundOpacity_0\\\\.1_mobile__3nw5tzu {
   --alpha__3nw5tz0: 0.1;
 }
-.styles_backgroundOpacity_0\\\\.2_mobile__3nw5tzx {
+.styles_backgroundOpacity_0\\\\.2_mobile__3nw5tzy {
   --alpha__3nw5tz0: 0.2;
 }
-.styles_backgroundOpacity_0\\\\.3_mobile__3nw5tz11 {
+.styles_backgroundOpacity_0\\\\.3_mobile__3nw5tz12 {
   --alpha__3nw5tz0: 0.3;
 }
-body > .styles__3nw5tz16 {
+.styles_color_red__3nw5tz16 {
+  --textAlpha__3nw5tz1: 1;
+  color: rgba(255, 0, 0, var(--textAlpha__3nw5tz1));
+}
+.styles_textOpacity_1__3nw5tz17 {
+  --textAlpha__3nw5tz1: 1;
+}
+.styles_textOpacity_0\\\\.8__3nw5tz18 {
+  --textAlpha__3nw5tz1: 0.8;
+}
+body > .styles__3nw5tz1a {
   background: red;
 }
 @media screen and (min-width: 768px) {
-  .styles_display_flex_tablet__3nw5tz2 {
+  .styles_display_flex_tablet__3nw5tz3 {
     display: flex;
   }
-  .styles_display_none_tablet__3nw5tz6 {
+  .styles_display_none_tablet__3nw5tz7 {
     display: none;
   }
-  .styles_display_block_tablet__3nw5tza {
+  .styles_display_block_tablet__3nw5tzb {
     display: block;
   }
-  .styles_paddingTop_small_tablet__3nw5tze {
+  .styles_paddingTop_small_tablet__3nw5tzf {
     padding-top: 10px;
   }
-  .styles_paddingTop_medium_tablet__3nw5tzi {
+  .styles_paddingTop_medium_tablet__3nw5tzj {
     padding-top: 20px;
   }
-  .styles_background_red_tablet__3nw5tzm {
+  .styles_background_red_tablet__3nw5tzn {
     --alpha__3nw5tz0: 1;
     background: rgba(255, 0, 0, var(--alpha__3nw5tz0));
   }
-  .styles_backgroundOpacity_1_tablet__3nw5tzq {
+  .styles_backgroundOpacity_1_tablet__3nw5tzr {
     --alpha__3nw5tz0: 1;
   }
-  .styles_backgroundOpacity_0\\\\.1_tablet__3nw5tzu {
+  .styles_backgroundOpacity_0\\\\.1_tablet__3nw5tzv {
     --alpha__3nw5tz0: 0.1;
   }
-  .styles_backgroundOpacity_0\\\\.2_tablet__3nw5tzy {
+  .styles_backgroundOpacity_0\\\\.2_tablet__3nw5tzz {
     --alpha__3nw5tz0: 0.2;
   }
-  .styles_backgroundOpacity_0\\\\.3_tablet__3nw5tz12 {
+  .styles_backgroundOpacity_0\\\\.3_tablet__3nw5tz13 {
     --alpha__3nw5tz0: 0.3;
   }
 }
 @media screen and (min-width: 1024px) {
-  .styles_display_flex_desktop__3nw5tz3 {
```

Comment:
I pushed an update to the snapshot, but it looks like it's removed some conditional styles?